code/__HELPERS/mobs.dm

```diff
@@ -486,3 +486,15 @@ GLOBAL_LIST_EMPTY(species_list)
 		chosen = pick(mob_spawn_meancritters)
 	var/mob/living/simple_animal/C = new chosen(spawn_location)
 	return C
+
+/proc/get_living_mobs()

```

Comment:
Misleading. Should be something like `get_living_station_minds()`.

Reply:
Actually it shouldn't have minds in it, it should have mobs. So the code needs to be changed.

Reply:
Made one for getting minds and one for getting mobs